frontend/public/components/dashboard/dashboards-page/cluster-dashboard/inventory-card.tsx

```diff
@@ -79,6 +79,7 @@ const ClusterInventoryItem = withDashboardResources<ClusterInventoryItemProps>(
             });
       }, [mapperLoader]);
 
+      // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Why was this needed?

Reply:
resolves
`The 'additionalResourcesData' object makes the dependencies of useCallback Hook (at line 105) change on every render. To fix this, wrap the initialization of 'additionalResourcesData' in its own useMemo()`

no need for useMemo here

Reply:
That sounds like a legitimate problem that shouldn't be ignored?